shell/app/node_main.cc

```diff
@@ -294,6 +296,16 @@ int NodeMain(int argc, char* argv[]) {
     node::ResetStdio();
 
     node::Stop(env);
+
+    // We do not want to invoke a termination exception at this stage when
+    // we're running with only_terminate_in_safe_scope set to false. Heap and
```

Comment:
> when we're running with only_terminate_in_safe_scope set to false

Is this always the case? Or should there be an `if() { ... }` wrapped around the `CancelTerminateExecution();` call?

Reply:
this is always the case - if we were in `javascript_environment` it'd be conditional but it will always be true when running in embedded Node.js mode